extras/userdoc/md/documentation/installation.md

```diff
@@ -450,34 +499,3 @@ please try making NEST as follows:
     env MACOSX_DEPLOYMENT_TARGET=10.5 make
 
 Thanks to Harold Gutch for this hint.
```

Comment:
Can we remove this? If not, I guess we would have to add many more people here...